tic-tac-toe-app/GameBoard.jsx

```diff
@@ -0,0 +1,69 @@
+function GameBoard() {
+  const [turn, setTurn] = React.useState(1);
+  const [cells, setCells] = React.useState([
+    null, null, null,
+    null, null, null,
+    null, null, null,
+  ]);
```

Comment:
This is definitely a valid way to set the default state. If you're up for a refactor, consider using another method to set an array to 9 null values. There are several alternatives, but look into [`Array.of()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/of) and [`Array.fill()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill) for starters.